Makefile

```diff
@@ -8,29 +8,10 @@ po:
 mo:
 	( cd allauth ; $(PYTHON) ../manage.py compilemessages )
 
-docs:
-	( . $(VIRTUALENV)/bin/activate; cd docs; make html )
-
-isort-check:
-	$(ISORT) -c
-
 isort-fix:
 	$(ISORT)
 
-pep8:
-	flake8 --exclude=migrations allauth
```

Comment:
Hmm, changes in Makefile looks as regression, why would you remove that lines?..